### PR TITLE
Fixes #13527: Error at install of Rudder Server Root 5.0 on Sles 12 

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/13527

### CODE REVIEW

**Comment by ncharles** [comment]
do not merge - it doesn't behave as expected on debian like system

**Comment by ncharles** [comment]
Commit modified

File: rudder-reports/SPECS/rudder-reports.spec [hunk 1]
```diff
@@ -111,7 +111,7 @@ install -m 644 %{SOURCE3} %{buildroot}/opt/rudder/etc/server-roles.d/
 # Post Installation
 #=================================================
 
-POSTGRESQL_SERVICE_NAME=$(systemctl list-unit-files --type service | awk -F'.' '{print $1}' | grep -E "^postgresql[0-9]*$" | tail -n 1)
+POSTGRESQL_SERVICE_NAME=$(systemctl list-units --type service | awk -F'.' '{print $1}' |  sed 's/[ ]*//g' | grep -E "^postgresql[0-9]*$" | tail -n 1)
```
Comment (peckpeck) [inline rudder-reports/SPECS/rudder-reports.spec]: why the [ ] ?

**Comment by ncharles** [comment]
Commit modified

**Pull request merged**